airbyte-workers/src/main/java/io/airbyte/workers/temporal/sync/RefreshSchemaActivityImpl.java

```diff
@@ -40,10 +44,15 @@ public boolean shouldRefreshSchema(UUID sourceCatalogId) throws IOException {
   }
 
   @Override
-  public void refreshSchema(UUID sourceCatalogId) throws ApiException {
+  public void refreshSchema(UUID sourceCatalogId, UUID connectionId) throws ApiException {
     SourceDiscoverSchemaRequestBody requestBody =
-        new SourceDiscoverSchemaRequestBody().sourceId(sourceCatalogId).disableCache(true);
-    sourceApi.discoverSchemaForSource(requestBody);
+        new SourceDiscoverSchemaRequestBody().sourceId(sourceCatalogId).disableCache(true).connectionId(connectionId);
+
+    try {
+      airbyteApiClient.getSourceApi().discoverSchemaForSource(requestBody);
+    } catch (final Exception e) {
+      log.info("Attempted schema refresh, but failed.");
```

Comment:
This should probably be an `error` log with the exception itself logged as well.

Reply:
oops yep, meant to update that! I also have a question about logging in general. These logs (and the ones in SyncWorkflowImpl) are not showing up in the UI logs and I'm not sure why.